lib/linguist/languages.yml

```diff
@@ -2450,6 +2453,17 @@ Go Module:
   tm_scope: go.mod
   ace_mode: text
   language_id: 947461016
+Go Workspace:
+  type: data
+  color: "#00ADD8"
+  aliases:
+  - go.work
+  - go work
+  filenames:
+  - go.work
+  tm_scope: go.work
```

Comment:
This won't work if a grammar doesn't provide this scope. As these files are structured like `go.mod` files so using that scope should do the trick.

Reply:
Oh, thank you for clarifying that! Will fix the scope ASAP